frameworks/PHP/cakephp/src/Controller/FortunesController.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+//App::uses('AppController', 'Controller');
+
+namespace App\Controller;
+
+use App\Model\Entity\Fortune;
+use Cake\ORM\TableRegistry;
+
+class FortunesController extends AppController {
+
+    public function index() {
+        $this->viewBuilder()->setLayout('fortunes');
+        $this->loadModel('Fortune');
+	$this->set('title_for_layout', 'Fortunes');
+
+        $fortunesTable = TableRegistry::getTableLocator()->get('Fortune');
+
+        $query = $fortunesTable->find('all');
+
+        // Calling all() will execute the query
+        // and return the result set.
+        $results = $query->all()->toArray();

```

Comment:
Please, use objects to have Full ORM.

> Using an ORM, all Fortune objects must be fetched from the Fortune table, and placed into a list data structure.

Reply:
Will do!

Reply:
Example for sorting the objects:
https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/PHP/yii2/app/controllers/SiteController.php#L52-L62